### DISCUSSION TITLE
So uh I'm doing a request...

### DISCUSSION THREAD

**Discussion by NX721** [Prodigy-Hacking/ProdigyMathGameHacking#796] [(C) Suggestions]
`_.instance.prodigy.education` Arcerion Told me to experiment for a question skip hack so yea.
You guys can help if ya want

**Comment by elikiller** [comment]
cool

**Reply by NX721** [reply]
Okay

**Comment by NX721** [comment]
@ArcerionDev  Apperantly `_.instance.prodigy.education = false;` despawns all monsters except Pippet etc.
![image](https://user-images.githubusercontent.com/77288637/112324456-330f9700-8c89-11eb-88d5-c934e68384ef.png)

**Comment by NX721** [comment]
And you don't interact with bosses, they speak their dialogue and then they just stand there and the game shows them but they aren't        in-game at the moment.

**Comment by NX721** [comment]
![image](https://user-images.githubusercontent.com/77288637/112324684-6c480700-8c89-11eb-83ce-46f062cdb44e.png)

**Comment by NX721** [comment]
And you're stuck.

**Comment by NX721** [comment]
And You lose the bottom menu

**Comment by ghost** [comment]
cool
